stubs/Drupal/datetime/Plugin/Field/FieldType/DateTimeItem.stub

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Drupal\datetime\Plugin\Field\FieldType;
+
+use Drupal\Core\Field\FieldItemBase;
+
+/**
+ * @property ?string $value
+ * @property-read ?\Drupal\Core\Datetime\DrupalDateTime $date
```

Comment:
Technically it isn't read only. Core itself is doing 

```
$this->date = NULL;
```

Can we change this? Maybe when these phpdoc improvements go into core we can fix usages

Reply:
Sure thing!